### PR TITLE
cdc: make tso worker threads configuable

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What is changed and how it works?

Issue Number: ref #12592

Add a configuration entry to custom cdc tso worker threads. With the config `cdc.min-ts-interval = 100ms`, test shows that 130K Region will utilize the thread about 70% usage. So single tso worker thread can be a bottle neck for some extreme cases.

By the way this PR also removes some noisy logs. And #12592 is also resolved.

```commit-message
 cdc: make tso worker threads configuable 
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -2416,6 +2423,7 @@ impl Default for CdcConfig {
             // is more than 500MB/s, so 128MB/s is enough.
             incremental_scan_speed_limit: ReadableSize::mb(128),
             incremental_scan_ts_filter_ratio: 0.2,
+            tso_worker_threads: 1,
```
Comment (zhangjinpeng1987) [inline src/config.rs:2426]: better to add a range limitation for its value.

Comment (zhangjinpeng1987) [reply src/config.rs:2426]: How to change it on the Cloud?

Comment (hicqu) [reply src/config.rs:2426]: Do you mean dynamically change it in TiDB? It's not available now.

**Review by overvenus — APPROVED** [review]

**Pull request merged**